ui/v2.5/src/models/list-filter/criteria/utils.ts

```diff
@@ -113,5 +113,17 @@ export function makeCriteria(type: CriterionType = "none") {
     case "piercings":
     case "aliases":
       return new StringCriterion(type, type);
+    case "url": {
+      const ret = new StringCriterion(type, type);
+      ret.modifierOptions = [
+        Criterion.getModifierOption(CriterionModifier.Equals),
+        Criterion.getModifierOption(CriterionModifier.NotEquals),
+        Criterion.getModifierOption(CriterionModifier.Includes),
+        Criterion.getModifierOption(CriterionModifier.Excludes),
+        Criterion.getModifierOption(CriterionModifier.MatchesRegex),
+        Criterion.getModifierOption(CriterionModifier.NotMatchesRegex),
+      ];
```

Comment:
Why is this necessary? The standard modifier options (which should include `IS [NOT] NULL`) should be sufficient.

Reply:
yes you are right, I removed it and use the standard modifier.